queries/rust/textobjects.scm

```diff
@@ -70,6 +70,7 @@
 (unsafe_block (_)? @block.inner) @block.outer
 
 ;; calls
+(macro_invocation) @call.outer
```

Comment:
Would it be possible to also add the corresponding `@call.inner` group? Maybe something like:

```scheme
(macro_invocation (token_tree . "(" . (_) @_start (_)? @_end . ")"
  (#make-range! "call.inner" @_start @_end)))
```

Reply:
Some basic testing using the query editor seems to validate this code; added in a commit.